deployment/start_prod.sh

```diff
@@ -3,14 +3,6 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd $SCRIPT_DIR
 
-echo "dropping old cme_* artifacts"
-echo "----------------------------------------"
-docker container rm cme_mongodb
-docker container rm cme_api
-docker network rm cme_internal
-echo ""
```

Comment:
Without this we generate a lot of dead containers which eat our already low storage capacity if I am not mistaken. Dropping the old state less parts of the infrastructure has also the benefit of changes going clearly through without having to drop them manually if something changes in the docker compose file.

Reply:
Oh, I am stupid and just saw the now used `down` command instead of the `stop` command. If it really works like it claims this seems smarter :)